CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.2.1] - 2017-07-19
```

Comment:
In many open source projects (including all the sensu ones) it's best to leave the versioning to the maintainers since there are no guarantees on the order in which they will be reviewed, merged, and released.